Makefile

```diff
@@ -29,6 +30,7 @@ OUTHEX_LP=$(LP).hex
 MAKEFILE=Makefile
 OUTDIR=BUILD
 OUTDIR_LP=BUILD_LP
+MSP430_TOOLCHAIN_PATH=/opt/msp430-gcc
```

Comment:
It is good to use `:=` such that an environment variable can still be used to override it.

Reply:
Yes, `:=`  or `?=` should be better.